src/components/hud/ActiveProjectDisplay.tsx

```diff
@@ -1,86 +1,68 @@
-import { CardContent, Typography } from '@mui/material'
+import { Button, Typography } from '@mui/material'
 import LinearProgress from '@mui/material/LinearProgress'
-import Card from '@mui/material/Card'
 import Grid from '@mui/material/Grid'
 import Input from '@mui/material/Input'
-import Slider from '@mui/material/Slider'
 import { projectCatalog } from '../../data/project-catalog'
-import { ActiveProject, WorkerState } from '../../game_logic'
+import { ActiveProject, ProjectDefinition, WorkerState } from '../../game_logic'
 import { dispatcher } from '../reducers'
+import { ProjectCard } from './ProjectCard'
 
-export function ActiveProjectDisplay (props: {activeProject: ActiveProject, tileIndex: number, workers: WorkerState, dispatch: dispatcher}) {
-  const projectDefinition = projectCatalog[props.activeProject.type]
-  const onWorkersChange = (event: any) => {
-    const workerCount = parseInt(event.target.value)
-    if (!isNaN(workerCount)) {
-      props.dispatch({ type: 'assignWorkers', tileIndex: props.tileIndex, workerCount })
-    }
-  }
-
+function activeProjectContent (props: {
+  activeProject: ActiveProject,
+  projectDefinition: ProjectDefinition,
+  onWorkersChange: (event: any) => void,
+  workers: WorkerState
+}) {
   const min = 0
-  const remainingEffort = projectDefinition.effort - props.activeProject.progress
+  const remainingEffort = props.projectDefinition.effort - props.activeProject.progress
   const max = Math.min(props.activeProject.assignedWorkers + props.workers.free, remainingEffort)
   const value = props.activeProject.assignedWorkers
-  const progress = props.activeProject.progress / projectDefinition.effort * 100
-  const bufferProgress = progress + value / projectDefinition.effort * 100
-
-  const grid2 = (
-    <Grid item xs>
-      <Input
-        value={value}
-        size='small'
-        onChange={onWorkersChange}
-        inputProps={{
-          step: 1,
-          min,
-          max,
-          type: 'number',
-          'aria-labelledby': 'input-slider'
-        }}
-      />
+  const progress = props.activeProject.progress / props.projectDefinition.effort * 100
+  const bufferProgress = progress + value / props.projectDefinition.effort * 100
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={12}><LinearProgress variant='buffer' value={progress} valueBuffer={bufferProgress} /></Grid>
+      <Grid item xs={8}><Typography>{props.projectDefinition.description}</Typography></Grid>
+      <Grid item xs={4}>
+        👤<Input
+          value={value}
+          size='small'
+          onChange={props.onWorkersChange}
+          inputProps={{
+            step: 1,
+            min,
+            max,
+            type: 'number'
+          }}
+          />
+      </Grid>
     </Grid>
   )
+}
 
-  const grid1 = (
-    <Grid item xs>
-      <Slider
-        value={typeof value === 'number' ? value : 0}
```

Comment:
I'd like to keep the slider, it's an easy input to use then typing a number

Reply:
I don't really agree, but if everyone feels strongly about the slider I can put it back in. Here are my thoughts:

- I noticed that you had a regular number input alongside the slider. I assume this is because the slider is imprecise, and so the regular number input is an escape hatch. To keep the UI simpler, I'd rather just have the input that always works.
- The number of workers can grow as contracts are completed. The discrete points on the slider will grow closer and closer together as this happens, making it even more difficult to get a precise number.

Playing around with it more I did notice a few bugs with the number input; I'll work on those for now, and if there is a consensus around the slider I can add it back.

Reply:
Does the number input have up/down arrows when it's focused?
I'd like there to be mouse only way to change number of workers.

Reply:
yeah the number input has the up/down arrows (I don't think that's browser dependent, either). the arrows are a bit smaller than I'd like, but they get the job done

For me, the ideal component [looks like this](https://i0.wp.com/codemyui.com/wp-content/uploads/2016/11/custom-input-field.gif?fit=880%2C440&ssl=1), with the text still editable using the keyboard, of course. I don't think we have the time to spend on it though, unfortunately